webpack/base.js

```diff
@@ -213,8 +200,7 @@ export function webpackDefaults(env, config, bundles, isPlugin = false) {
           sourceMap: true,
           sassOptions: {
             includePaths: [scssIncludePath],
-            outputStyle: 'compact',
-            importer: [sassResolver, _importOnce],
```

Comment:
Good point, I'll probably have to switch from `@import` to `@use/@forward`

Reply:
I think individual commits will be more helpful here than separate PRs here.... That way it's just one PR to test with if nothing breaks.

Reply:
FWIW "separating library stuff" would certainly deserve a separate PR. "avoiding duplication" belongs in here.

Reply:
Yeah that's what I meant, we can have a separate PR to consolidate the library stuff. Duplication should not be an issue with `@use` since it imports each file only once.